library/notation-issuer-validation/template.yaml

```diff
@@ -21,7 +21,11 @@ spec:
         # Get data from Ratify
         remote_data := response {
           images := [img | img = input.review.object.spec.containers[_].image]
-          response := external_data({"provider": "ratify-provider", "keys": images})
+          images_init := [img | img = input.review.object.spec.initContainers[_].image]
+          images_ephemeral := [img | img = input.review.object.spec.ephemeralContainers[_].image]
+          other_images := array.concat(images_init, images_ephemeral)
+          all_images := array.concat(other_images, images)
```

Comment:
re: @binbin-li's comment about dedup:

That's a great point. The `/verify` handler spawns a go routine per subject image and a lock per unique subject reference is acquired. So even if there are duplicates in the list, this shouldn't result in extra processing on Ratify's end. Dedup will likely be better although not sure how much work that is. I'm ok either way (dedup or not)